plugins/in_winevtlog/in_winevtlog.c

```diff
@@ -28,7 +28,10 @@
 
 #define DEFAULT_INTERVAL_SEC  1
 #define DEFAULT_INTERVAL_NSEC 0
-#define DEFAULT_THRESHOLD_SIZE 0x7ffff /* Default reading buffer size (512kb) */
+#define DEFAULT_THRESHOLD_SIZE 0x7ffff /* Default reading buffer size */
+                                       /* (512kib = 524287bytes) */
+#define MINIMUM_THRESHOLD_SIZE 0x0400   /* 1024 bytes */
+#define MAXIMUM_THRESHOLD_SIZE 0x1ccccd /* 1887437 bytes (about 1.8 MiB) */
```

Comment:
Set `MAXIMUM_THRESHOLD_SIZE` to `(FLB_INPUT_CHUNK_FS_MAX_SIZE - (1024 * 200))` so it scales in case someone changes the default maximum chunk size. 

I guess in the future when we make the chunk size configurable we'll have to do that in runtime.

Reply:
Sorry about not requesting it this way the first time around, it must've slipped from my mind.

Reply:
No worries. Your suggestion is also helpful. And it has more effective than just specified the fixed value for the current adequate value.